build/OutputDirectories.props

```diff
@@ -7,6 +7,7 @@
       <DotnetStage1>$(Stage1Directory)/dotnet$(ExeExtension)</DotnetStage1>
       <Stage2Directory>$(BaseOutputDirectory)/stage2</Stage2Directory>
       <Stage2SymbolsDirectory>$(BaseOutputDirectory)/stage2symbols</Stage2SymbolsDirectory>
+      <Stage2SdkDirectory>$(Stage2Directory)/sdk/$(CommitCount)</Stage2SdkDirectory>
```

Comment:
Should be able to remove this duplication in the Compile.targets file by changing this line:

'<SdkOutputDirectory>$(StageDirectory)/sdk/$(SdkVersion)</SdkOutputDirectory>'

to

'<SdkOutputDirectory>$(Stage2SdkDirectory)</SdkOutputDirectory>'